dev-tools/mage/common.go

```diff
@@ -821,3 +821,28 @@ func ListMatchingEnvVars(prefixes ...string) []string {
 	}
 	return vars
 }
+
+// CloudEnvVars returns the names of environment variables needed to configure
+// connections to cloud environments.
+func CloudEnvVars() []string {
```

Comment:
This could also include others like KUBECONFIG in the future right? I wonder if it should just be called IntegrationTestEnvVars or similar

Reply:
:+1: renamed.